indra/newview/skins/default/xui/en/floater_im_session.xml

```diff
@@ -291,6 +291,7 @@
                          height="20"    
                          is_expandable="true"
                          text_tentative_color="TextFgTentativeColor"
+                         bg_writeable_color="ScriptBackground"
```

Comment:
This will work for everything, but shouldn't it be not turning grey only for emoji flaoter?